ow_faults/src/FaultInjector.cpp

```diff
@@ -259,32 +172,18 @@ void FaultInjector::jointStateCb(const sensor_msgs::JointStateConstPtr& msg)
     output.effort[index]  = FAULT_ZERO_TELEMETRY;
   }
 
-  if (m_arm_fault) {
-    m_system_faults_bitset |= isArmExecutionError;
-    setComponentFaultsMessage(arm_faults_msg, hardwareFault);
-  } else {
-    m_system_faults_bitset &= ~isArmExecutionError;
-  }
-
-  if (m_cam_fault){
-    m_system_faults_bitset |= isCamExecutionError;
-    setComponentFaultsMessage(camera_faults_msg, hardwareFault);
-  } else {
-    m_system_faults_bitset &= ~isCamExecutionError;
-  }
+  // if (m_arm_fault) {
+  //   m_system_faults_bitset |= isArmExecutionError;
+  //   setComponentFaultsMessage(arm_faults_msg, hardwareFault);
+  // } else {
+  //   m_system_faults_bitset &= ~isArmExecutionError;
+  // }
 
   m_joint_state_pub.publish(output);
-  publishSystemFaultsMessage();
-
-  m_arm_fault_msg_pub.publish(arm_faults_msg);
-  m_antenna_fault_msg_pub.publish(pt_faults_msg);
-  m_camera_fault_msg_pub.publish(camera_faults_msg);
-}
+  // publishSystemFaultsMessage();
 
-void FaultInjector::publishSystemFaultsMessage(){
-  ow_faults::SystemFaults system_faults_msg;
-  setBitsetFaultsMessage(system_faults_msg, m_system_faults_bitset);
-  m_system_fault_msg_pub.publish(system_faults_msg);
+  // m_arm_fault_msg_pub.publish(arm_faults_msg);
+  // m_antenna_fault_msg_pub.publish(pt_faults_msg);
```

Comment:
For all commented-out code, either remove if no longer needed, or add comment explaining why it's left in.